bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html

```diff
@@ -8,31 +8,778 @@
+    to verify certificates for SSL servers and S/MIME email users without having
+    to ask users for further permission or information.</p>
+  <p>This is the official Mozilla policy for CA certificates that are distributed
+    with Mozilla software products. This policy consists of the following three
+    sections:</p>
+  <ul>
+    <li>
+      <p><a href="#inclusion">Applying for Inclusion of Root Certificates in Mozilla Products</a>
+      </p>
+      <p>This section describes the obligations of Certification Authorities
+        applying for inclusion of their root certificates in Mozilla Products.
+        This includes considerations that are taken into account such as the CA’s
+        publicly available documentation about their policies, and audits of the
+        CA’s operations in support of the documented policies.</p>
+    </li>
+    <li>
+      <p><a href="#maintenance">Maintaining Confidence in Included Root Certificates</a></p>
+      <p>This section describes the obligations of Certification Authorities for
+        maintaining confidence in their root certificates that are included in
+        Mozilla Products. This includes regular auditing of the CA’s policies and
+        practices; conforming to current CA industry standards and recommended
+        best practices; and making changes to included root certificates.</p>
+    </li>
+    <li>
+      <p><a href="#enforcement">Enforcing the Mozilla CA Certificate Policy</a></p>
+      <p>This section describes the steps that Mozilla may take in order to enforce
+        this policy. This includes evaluation of security concerns, and removing
+        or disabling a root certificate.</p>
+    </li>
+  </ul>
+  <p>This policy applies only to software products distributed by Mozilla,
+    including the Mozilla Foundation and its subsidiaries. Other entities
+    distributing such software are free to adopt their own policies. In
+    particular, under the terms of the relevant Mozilla license(s) distributors
+    of such software are permitted to add or delete CA certificates in the
+    versions that they distribute, and are also permitted to modify the values of
+    the "trust bits" on CA certificates in the default CA certificate set. As with
+    other software modifications, by making such changes a distributor may affect
+    its ability to use Mozilla trademarks in connection with its versions of the
+    software; see the Mozilla trademark policy for more information.</p>
+  <p>Please contact Mozilla at
+    <a href="mailto:certificates@mozilla.org">certificates@mozilla.org</a> for more
+    information about this policy and answers to related questions.</p>
+  <p>We reserve the right to change this policy in the future. We will do so only
+    after consulting with the public Mozilla community, in order to ensure that
+    all views are taken into account.</p>
+  <p><a name="inclusion"></a></p>
```

Comment:
Nit: named anchors should be avoided these days. It'll still work in browsers but it's not valid in HTML5 and can present accessibility problems. This should be an ID on the `h2` below to provide an anchor for linking: `<h2 id="inclusion">`